novuhq/website
# chore: update the text color

**Describe what changes this pull request brings**
This pull request adds updates colors for text making it more readable.

**Steps to test**

1. Open [the page](https://websitemain13551-updatecolortext.gtsb.io/)
2. Confirm that everything looks and works as expected

**Screenshots**
Before:
<img width="1581" alt="Screenshot 2022-07-06 at 13 22 43" src="https://user-images.githubusercontent.com/17677196/177529594-b0e9b239-9999-4afc-8328-a1f16c2eeb7d.png">

After:
<img width="1574" alt="Screenshot 2022-07-06 at 13 24 01" src="https://user-images.githubusercontent.com/17677196/177529638-94272c95-dc64-47d4-b810-b430f1252709.png">

Before:
<img width="1542" alt="Screenshot 2022-07-06 at 13 22 50" src="https://user-images.githubusercontent.com/17677196/177529679-a7a1cabb-6e63-4016-beb7-f4caa830b7c1.png">


After:
<img width="1526" alt="Screenshot 2022-07-06 at 13 24 13" src="https://user-images.githubusercontent.com/17677196/177529732-e2242282-df70-40d8-b83a-9aacbcea5bc7.png">
